index.js

```diff
@@ -0,0 +1,72 @@
+const defaultOptions = {
+    styles: '*:focus  { outline: none !important; }',
+    stylesheetTarget: document.head,
+};
+
+const acceptedKeys = { Tab: 9, ArrowLeft: 37, ArrowTop: 38, ArrowRight: 39, ArrowDown: 40 };
+
+const createStylesheet = (options) => {
+    const styleNode = document.createElement('style');
+    const targetNode = options.stylesheetTarget;
+
+    styleNode.innerText = options.styles;
+
+    return {
+        apply: () => {
+            targetNode.append(styleNode);
+        },
+        unapply: () => {
+            if (styleNode.parentNode === targetNode) {
+                targetNode.removeChild(styleNode);
+            }
+        },
+    };
+};
+
+const createListeners = (styleSheet) => {
+    let outlinesEnabled = true;
+
+    const handleKeydown = (ev) => {
```

Comment:
I think the implementation could be improved:

1. `Object.values` will suffice to pull the key codes out of `acceptedKeys` object in array form in a single pass, rendering `Object.keys` unnecessary here. Furthermore, you are passing an array to `Object.values`, which is not correct. An object must be passed.

2. Unless `acceptedKeys` can change in runtime, which seems not to be the case, there is no need to recompute `Object.values` every time the `keydown` listener is fired. We can cache the result.

What do you think?

Reply:
That `Object.values(acceptedKeyCodes)` was not supposed to be there. My bad. The reason I decided to use `Object.keys` in detriment of `Object.values` is because the former is supported in IE11, while the latter is not.